16/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/rich-text-editor/blocks.md

```diff
@@ -0,0 +1,212 @@
+
+![RTE Insert Block Toolbar Button](images/rte-blocks-toolbar-insert-button.png)
+
+## Configure Blocks
+
+Blocks are Element Types that must be created before configuring them as blocks in the Rich Text Editor. You can create Element Types in the **Settings** > **Document Types** section
+
+Blocks functionality can then be configured through the Rich Text Editor Data Type configuration as follows.
+
+1. Navigate to **Settings** > **Data Types**.
+2. Select your Rich Text Editor Data Type or create a new one.
+3. In the **Available Blocks** section, add the Element Types you want to make available as blocks.
+4. Configure each Block Type with the options described below.
+
+![RTE Blocks Data Type Configuration](images/rte-blocks-datatype-configuration.png)
+
+### Editor Appearance
+
+Configure how blocks appear and behave in the Content section:
+
+*   **Label** - Define how the block appears in the editor. Umbraco 16 uses [Umbraco Flavoured Markdown](../../../../../reference/umbraco-flavored-markdown.md) (UFM) syntax for dynamic labels. Use `{=propertyAlias}` to display property values (e.g., `{=author}` for a text property containing the name of an author, or `Written by: {=author}` for a label with static text and dynamic content)
+*   **Display Inline** - When enabled, blocks remain inline with surrounding text. When disabled, blocks appear on separate lines
+*   **Overlay size** - Set the size of the editing overlay when editors work with the block content
+
+![RTE Blocks Editor Appearance Settings](images/rte-blocks-editor-appearance-settings.png)
+
+### Data Models
+
+Configure the content structure for your blocks:
+
+*   **Content model** - The Element Type that defines the main content properties for the block (required)
+*   **Settings model** - Optional Element Type that defines additional settings or configuration options for the block
+
+![RTE Blocks Data Models Settings](images/rte-blocks-data-models-settings.png)
+
+### Catalogue Appearance
+
+Control how blocks appear in the block picker:
+
+*   **Background color** - Background color displayed behind the block icon or thumbnail
+*   **Icon Color** - Color of the Element Type icon
+*   **Thumbnail** - Custom image to replace the default Element Type icon
+
+## Working with Blocks
+
+### Adding Blocks to Content
+
+Editors can add blocks to rich text content using the **Insert Block** toolbar button:
+
+1. Position the cursor where you want to insert the block
+2. Click the **Insert Block** button in the Rich Text Editor toolbar
+3. Select the desired Block from the available options
+4. Configure the block content (and settings, if provided)
+5. The block appears in the editor as a structured element
+
+![Adding Blocks to Content - Step 1](images/rte-blocks-adding-to-content-1.png)
+
+![Adding Blocks to Content - Step 2](images/rte-blocks-adding-to-content-2.png)
+
+![Adding Blocks to Content - Step 3](images/rte-blocks-adding-to-content-3.png)
```

Comment:
Could you add these images to the list instead?
That way you see what you need to do, when you need to do it.

Add them to the list by following this format:

1. Instruction.

![image text](image.png)

2. More instructions.

Hope it makes sense 🙏